crates/terminal_view/src/terminal_view.rs

```diff
@@ -418,21 +420,6 @@ impl TerminalView {
         .detach();
     }
 
-    pub fn find_matches(
```

Comment:
This seems unused or am I missing something?

Reply:
I think it's us who missed that after a bit gpui -> gpui2 rewrite, and you're right.